arangod/Agency/AgencyStrings.h

```diff
@@ -50,6 +50,8 @@ constexpr char const* PLAN_DATABASES = "Plan/Databases/";
 constexpr char const* PLAN_VIEWS = "Plan/Views/";
 constexpr char const* PLAN_ANALYZERS = "Plan/Analyzers/";
 
+constexpr char const* HOTBACKUP_TRANSFER_LOCKS = "/Target/Hotbackup/Transfers/";
```

Comment:
Hotbackup --> HotBackup?

Reply:
No, we have unfortunately `/arango/Target/Hotbackup/Transfers` and `/arango/Target/HotBackup/TransferJobs` and in 3.8 I will not change this for backward compatibility reasons. In 3.9 and devel, I will change this and make it consistent. I will also use URLEncode in the agency paths.